src/main/java/org/cryptomator/ui/vaultoptions/VaultOptionsController.java

```diff
@@ -48,6 +49,11 @@ public void initialize() {
 		if(!(vaultScheme.equals(HubKeyLoadingStrategy.SCHEME_HUB_HTTP) || vaultScheme.equals(HubKeyLoadingStrategy.SCHEME_HUB_HTTPS))){
 			tabPane.getTabs().remove(hubTab);
 		}
+
+		// Fixes: https://github.com/cryptomator/cryptomator/pull/3267
```

Comment:
remove the comment. The reason for this code will be clear vom version history 😉

Reply:
Just did